src/renderer/vt/paint.cpp

```diff
@@ -155,6 +155,11 @@ using namespace Microsoft::Console::Types;
     return S_OK;
 }
 
+[[nodiscard]] HRESULT VtEngine::PaintTerminalEffects() noexcept

```

Comment:
This needs to be implemented on the `BgfxEngine` and the `WddmEngine` as well (which are in `/src/interactivity/onecore` and `/src/renderer/wddmcon` respectively but not actually projects in the solution file.)

Reply:
Removing this from the interface